distributed/sizeof.py

```diff
@@ -62,3 +62,18 @@ def sizeof_pandas_index(i):
         p = int(i.memory_usage())
         obj = len(i) * 100 if i.dtype == object else 0
         return int(p + obj) + 1000
+
+
+@sizeof.register_lazy("scipy")
+def register_spmatrix():
+    from scipy import sparse
+    @sizeof.register(sparse.dok_matrix)
+    def sizeof_spmatrix_dok(s):
+        return s.__sizeof__()
+
+    @sizeof.register(sparse.spmatrix)
+    def sizeof_spmatrix(s):
+        return sum(
+            sizeof(v) for v in s.__dict__.values()
+        )
```

Comment:
How slow is this in the common case?  Are these values all of a certain dtype?  If so then are there other ways to compute this more quickly?  For example by returning `len(s.__dict__) * len(first(s.values()))`

Also in cases like this it might be nicer to put this code on one line if it fits.